package.json

```diff
@@ -18,6 +18,8 @@
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-typescript": "^7.3.3",
     "@firebase/testing": "^0.16.7",
+    "@react-native-firebase/app": "^6.4.0",
+    "@react-native-firebase/firestore": "^6.4.0",
```

Comment:
As parts of those libs are actually used in the code, maybe those could be `peer` dependencies instead?

Reply:
Initially, I had these in the peer dependencies, but it turned to be impractical as purely Node.js projects would get complaints about missing the `firebase` module and vice versa.  Now with three different environments, it would even worse.

Reply:
Got it, maybe it'd be good to loosen a bit packages version. I'm not NPM expert, but seems like if I'd use very different version of those - I might end up having 2 versions of firebase installed in the app and in this module, which could lead to issues in dev mode

I'm not sure, however